src/middlewares/auth/auth.js

```diff
@@ -0,0 +1,29 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+export const authorization = (req, res, next) => {
+  const header = req.header('Authorization');
+
+  if (typeof header !== 'undefined') {
+    const bearer = header;
+    const token = bearer;
```

Comment:
Why assign bearer to header then token to bearer?

Reply:
I extracted the token first from the header by splitting it to bearer and token